packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/StakingDashboard.tsx

```diff
@@ -62,6 +91,30 @@ export const StakingDashboard = () => {
         selectAccountUnstakeTransactions(state, account?.key ?? ''),
     );
 
+    const dispatch = useDispatch();
```

Comment:
can you move this logic move to InstantStakeBanner component? this component is getting big

Reply:
sure, 6a6d33b28e105a1f2edf2da7a7e2fe1256ecf019